__tests__/app.test.js

```diff
@@ -190,4 +190,42 @@ describe("app ", () => {
         .expect(400);
     });
   });
+  describe("PATCH /api/reviews/:review_id", () => {
+    test("increment votes by 1 responds with the update review ", () => {
+      return request(app)
+        .patch("/api/reviews/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then((updatedReview) => {
+          const reviewData1 = testData.reviewData[0].votes;
+          expect(reviewData1).not.toBe(updatedReview.body.votes);
+          expect(updatedReview.body.votes).toBeGreaterThan(reviewData1);
+          expect(updatedReview.body.votes).toBe(2);
+        });
```

Comment:
If you wanted to be thorough you could also add a test that just checks the returned comment object is the shape you expect, (i.e. has all the correct keys) as you've only check the increased votes 🔧